akka-persistence/src/test/scala/akka/persistence/PersistentActorSpec.scala

```diff
@@ -24,10 +22,10 @@ object PersistentActorSpec {
   final case class LatchCmd(latch: TestLatch, data: Any) extends NoSerializationVerificationNeeded
   final case class Delete(toSequenceNr: Long)
 
-  abstract class ExamplePersistentActor(name: String) extends NamedPersistentActor(name) with PersistentActor {
+  abstract class ExamplePersistentActor(name: String) extends NamedPersistentActor(name) {
     var events: List[Any] = Nil
 
-    val updateState: Receive = {
+    lazy val updateState: Receive = {
```

Comment:
doesn't need to, sorry - that's a leftover from hunting "WHY NPE" in one of the refactors I did.